quarkchain/cluster/master.py

```diff
@@ -994,8 +994,24 @@ def log_error_and_exit(msg):
 
             for crb in rb_list:
                 try:
+                    # lower the difficulty for root block signed by guardian
```

Comment:
Could we put a common code for master so that we have a single code path in master to work on root chain posw?

Reply:
sounds good. addressed in #753